CHANGELOG.md

```diff
@@ -18,6 +25,7 @@ All notable changes to this project will be documented in this file.
 - Bumped invoca-utils to 0.3.0
 
 ## [1.2.1] - 2020-04-27
+### Changed
```

Comment:
💯 
That's quite a few rookie mistakes you had to clean up here in the CHANGELOG. 😛